pkg/trace/obfuscate/sql.go

```diff
@@ -195,20 +223,24 @@ type tableFinderFilter struct {
 // Filter implements tokenFilter.
 func (f *tableFinderFilter) Filter(token, lastToken TokenKind, buffer []byte) (TokenKind, []byte, error) {
 	switch lastToken {
-	case From:
+	case From, Join:
```

Comment:
Why was the `Join` moved?

Reply:
You can join on a subquery. e.g. `JOIN (SELECT * FROM table)`

Reply:
Good point! Thanks! Resolving